bundletester/builder.py

```diff
@@ -108,14 +109,22 @@ def reset(self):
                     )
                     break
                 except Exception as e:
-                    logging.exception(e)
-
+                    reconnect_errors = [
+                        errno.ETIMEDOUT,
+                        errno.EPIPE,
+                        errno.ECONNABORTED,
+                        errno.ECONNRESET,
+                        errno.ENETRESET,
+                    ]
                     if isinstance(
-                            e, websocket.WebSocketConnectionClosedException):
-                        logging.debug('Reconnectinng to environment...')
+                            e, websocket.WebSocketConnectionClosedException) \
+                       or getattr(e, 'errno', None) in reconnect_errors:
```

Comment:
Update the indentation on the above line:

```
                    if isinstance(
                            e, websocket.WebSocketConnectionClosedException) \ 
                            or getattr(e, 'errno', None) in reconnect_errors:
                        logging.debug('Reconnecting to environment...')
                        self.environment.connect()
                        continue
```

Also, parentheses should be used in preference to using a backslash for line continuation https://www.python.org/dev/peps/pep-0008/#maximum-line-length